src/domains/main/common/components/BestList.jsx

```
@@ -24,9 +24,9 @@ const BestList = ({ boardName, bestList }) => {
         </video>
         <div className='text-xl font-bold sm:text-3xl'>{boardName} 베스트</div>
       </div>
-      <div className='grid grid-cols-1 sm:grid-cols-2 gap-y-4 sm:gap-x-20 sm:grid-flow-col sm:grid-rows-5'>
-        {bestList.length !== 0 ? (
-          bestList.map(({ title, boardId, jobPostingId, commentCount, viewCount }, index) => {
+      {bestList.length !== 0 ? (
```

ESLint

[error] 'bestList.length' is missing in props validation

(react/prop-types)